requirements.txt

```diff
@@ -23,3 +23,5 @@ toml==0.10.2
 urllib3==1.26.3
 webencodings==0.5.1
 Werkzeug==1.0.1
+flask_api
```

Comment:
Did you use `pip freeze > requirements.txt`? The dependencies should also be locked to a version number.

Reply:
Why doesn't `flask_api` have a version number?

Reply:
😒 Don't ask me how I missed that, fixing now

Reply:
[Resolved for the final time](https://github.com/BoringStocks/StonkScraper/pull/12/commits/1733617c5629a5fe89cc7b29245ac3c157295121)